### ISSUE TITLE
List Response Names

### ISSUE THREAD

**Issue by PeterJFerrarotto** [JamesMarcogliese/slack-api-csharp#3]
When calling a List function, the class names are as follows:
ListOKResponseModel
ListOKResponseModelModel
ListOKResponseModelModelModel...
It carries on until there are nine "Model"s at the end.

Obviously not critical, but it would be better to have actual names corresponding to the action that returns them rather than something so arbitrary.

**Comment by JamesMarcogliese** [comment]
I agree 100%. Wasn't until I was looking at issue #2 that I realised the generator spat them out like so. I'll start converting them this weekend as I move through. I'll also be adding the SlackAttachment class I created to the EventApi folder.
